packages/rocketchat-statistics/server/functions/getUsages.js

```diff
@@ -29,7 +29,7 @@ export function getUsages() {
 		{
 			$group:
 				{
-					_id: { uid: '$u._id', subType: '$t'},
+					_id: { uid: '$u._id', subType: {$cond: [{$not: ['$parentRoomId']}, '$t', 't']}},
```

Comment:
I didn't know `$cond` - and particularly `'$t', 't'` looks really weird.
I now understand and it looks good to me, but you could really add some comment, e. g. 
`// we'll group by roomtype, unless the room has a parentRoomId. In this case, we'll assign a virtual room type t for "thread"`